frontend/@types/console/index.d.ts

```diff
@@ -46,6 +46,7 @@ declare interface Window {
     consolePlugins: string[]; // Console dynamic plugins enabled on the cluster
     quickStarts: string;
     projectAccessClusterRoles: string;
+    multicluster: boolean;
```

Comment:
We'll want to align with how we test this in #9340. Probably you'll want to reuse `isMultiClusterEnabled`.

https://github.com/openshift/console/pull/9340/files#diff-52414ab87aedb2a2b855ef5f11c0d85c63286f32c74831d1edb730919c3fc220R1

cc @abhinandan13jan @christianvogt

Reply:
Is there a case where ACM is installed, but there is only one cluster?  https://github.com/openshift/console/pull/9340/files#diff-5f2cc074d9cdf86072a6e8caf63290f7d49b2e126b2c30ff3159bd36923dfff6R9 

Or does ACM always require more than one cluster?

I ask as the switching in this PR is specific to the presence of ACM and not necessarily multicluster.

Reply:
@rhamilto I think you could check for an empty array, which would mean the ACM integration is not configured. If ACM has no managed clusters, you'd get a list of size one with `local-cluster`.

@TheRealJon can you confirm?

Reply:
We haven't implemented the back end changes yet, but I can make it so.

Reply:
@spadgett i thought you said that we'd always at least have a single item in the array. If with ACM integration or not, we'd have `local-cluster` present.
I think having a `null` value is fine to with no ACM integration though

Reply:
Looking to close the loop on this...  What is the correct check for ACM configuration?   I am unclear on what the correct value of `window.SERVER_FLAGS.clusters` will be.  Is `local-cluster` always present in the array or not?

Reply:
IMO, `window.SERVER_FLAGS.clusters` should always contain the cluster that the console is running on, so it would never be empty.